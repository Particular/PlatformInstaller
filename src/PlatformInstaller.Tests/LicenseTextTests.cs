using ApprovalTests;
using NUnit.Framework;

[TestFixture]
public class LicenseTextTests
{
    [Test]
    public void VerifyLicenseHtml()
    {
        Approvals.Verify(LicenseText.ReadLicenseHtml(),s => s.Replace("\r\n","\n"));
    }
}