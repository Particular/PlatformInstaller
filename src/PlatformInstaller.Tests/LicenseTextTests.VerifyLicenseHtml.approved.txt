<!DOCTYPE html>
<html lang='en' xmlns='http://www.w3.org/1999/xhtml'>
<head>
<meta charset='utf-8' />
<meta http-equiv='X-UA-Compatible' content='IE=edge' /> 
    <style type='text/css'>
        BODY 
        {
            font-size: 9pt !important;
            color: #000 !important;
            font-family: 'Segoe UI' !important;
        }
    </style>
</head>
<body><h3 id="nservicebus-evaluation-license-agreement">NSERVICEBUS EVALUATION LICENSE AGREEMENT</h3>
<p>IMPORTANT - READ CAREFULLY THE TERMS AND CONDITIONS OF THIS LICENSE AGREEMENT (THIS &quot;AGREEMENT&quot;) BEFORE CLICKING ON THE &quot;I AGREE&quot; BUTTON. BY CLICKING ON THE &quot;I AGREE&quot; BUTTON, INSTALLING OR USING ALL OR ANY PORTION OF THE SOFTWARE (AS DEFINED BELOW), YOU, ON YOUR OWN BEHALF AS AN INDIVIDUAL, AND ON BEHALF OF YOUR EMPLOYER OR ANOTHER ENTITY WHICH HAS NOT YET DONE SO (COLLECTIVELY &quot;YOU&quot;, OR &quot;LICENSEE&quot;),ACKNOWLEDGE THAT YOU HAVE READ THIS AGREEMENT, THAT YOU UNDERSTAND IT, AND ARE CONSENTING TO BE BOUND BY AND ARE BECOMING A PARTY TO THIS AGREEMENT WITH NSERVICEBUS LTD. (DOING BUSINESS AS PARTICULAR SOFTWARE) (&quot;NSB&quot;). IF YOU DO NOT AGREE TO THE TERMS OF THIS AGREEMENT, YOU MAY NOT DOWNLOAD, INSTALL OR USE NSB&#39;S SOFTWARE AND RELATED DOCUMENTATION. IF YOU WISH TO USE THE SOFTWARE AS AN EMPLOYEE, CONTRACTOR, OR AGENT OF A CORPORATION, PARTNERSHIP OR SIMILAR ENTITY, THEN YOU MUST BE AUTHORIZED TO SIGN FOR AND BIND THE ENTITY IN ORDER TO ACCEPT THE TERMS OF THIS AGREEMENT. THE LICENSE GRANTED UNDER THIS AGREEMENT IS EXPRESSLY CONDITIONED UPON ACCEPTANCE BY SUCH AUTHORIZED PERSONNEL. IF YOU HAVE ENTERED INTO A SEPARATE WRITTEN LICENSE AGREEMENT WITH NSB FOR USE OF THE SOFTWARE, THE TERMS AND CONDITIONS OF SUCH OTHER AGREEMENT SHALL PREVAIL OVER ANY CONFLICTING TERMS OR CONDITIONS IN THIS AGREEMENT.</p>
<p>FURTHERMORE, YOU HEREBY WAIVE ANY RIGHTS OR REQUIREMENTS UNDER ANY LAWS OR REGULATIONS IN ANY JURISDICTION WHICH REQUIRE AN ORIGINAL (NON-ELECTRONIC) SIGNATURE OR DELIVERY OR RETENTION OF NON-ELECTRONIC RECORDS, TO THE EXTENT PERMITTED UNDER APPLICABLE LAW.</p>
<p>YOU ASSUME ALL RESPONSIBILITY FOR THE SELECTION OF THE SOFTWARE TO ACHIEVE YOUR INTENDED RESULTS AND FOR THE INSTALLATION, USE AND RESULTS OBTAINED FROM THE SOFTWARE.</p>
<ol>
<li><strong>Evaluation Period</strong>: 14 days from the date Licensee&#39;s installs or receives access to the Software as may be extended by NSB at its sole discretion (&quot;<b>Evaluation Period</b>&quot;).</li>
<li><strong>Software to be Evaluated</strong>: the Particular Service Platform (the &quot;<b>Software</b>&quot;).</li>
<li><strong>License Grant</strong>: During the Evaluation Period, NSB hereby grants to Licensee a temporary, nonexclusive, revocable, nontransferable, limited license to use the Software, solely to evaluate whether to license the Software from NSB or enter into some other commercial arrangement with NSB, and only as authorized in this Agreement.  Licensee may not use the Software for general production use or following the Evaluation Period. The Software is licensed for Licensee’s internal use, and neither the Software nor any derivative of the Software may be used by, sub-licensed, re-sold, rented or distributed to any other party.</li>
<li><strong>License Restrictions:</strong> Except as expressly and unambiguously permitted by this Agreement, Licensee may not, nor permit anyone else to, directly or indirectly: (i) commercially distribute, rent, lease, market, sublicense, resell or otherwise transfer the Software, (ii) copy or modify the Software either alone or in conjunction with any other product or program, (iii) remove any identification, including copyright, trademark, patent or other notices, contained in or on the Software; or (vi) use the NSB name, logo or trademarks without prior written consent from NSB.</li>
<li><strong>Ownership Rights:</strong> Licensee acknowledges and agrees that all right, title, and interest in and to the Software, including associated intellectual property rights, are and shall remain with NSB. This Agreement does not convey to Licensee any ownership interest in the Software.</li>
<li><strong>Term of License:</strong> Licensee’s license to use the Software will terminate upon expiration of the Evaluation Period. Unauthorized copying of the Software or otherwise failing to comply with this Agreement will result in automatic immediate termination of this Agreement and will make available to NSB legal remedies. Upon termination of the license Licensee agrees to uninstall and cease using the Software.</li>
<li><strong>Nondisclosure:</strong> Licensee agrees not to disclose any portion of the Software to any third party and to take reasonable steps to ensure the Software is not disclosed or distributed by Licensee’s employees or agents in violation of the provisions of this Agreement.</li>
<li><strong>Feedback:</strong> Without derogating from the aforesaid, Licensee hereby grant NSB an irrevocable, free of charge, license under all intellectual property rights (including copyright) to use, for any and all purposes, any feedback, including materials, error corrections, enhancements, modifications, derivatives, suggestions and the like that Licensee may provide NSB.</li>
<li><strong>Third-Party Software:</strong> Licensee acknowledges that portions of the Software include Third-Party Software (including open source software) that may be subject to the terms and conditions imposed by the licensors of the Third-Party Software (&quot;<b>Third-Party Terms</b>&quot;).  In order to comply with the Third Party Terms a list of the Third Party-Software and related Third-Party Terms is available at <a href="http://particular.net/ThirdPartyLicenses">http://particular.net/ThirdPartyLicenses</a>. Licensee agrees (i) that its use of the Third-Party Software is subject to and governed by the Third-Party Terms; and (ii) to comply with all Third-Party Terms.  To the extent of any conflict between any Third-Party Terms and the terms of this Agreement, the Third-Party Terms shall prevail in connection with the related Third-Party Software.  Notwithstanding anything to the contrary in this Agreement, NSB makes no warranty or indemnity with respect to any Third-Party Software.</li>
<li><strong>Exclusion of Warranty:</strong> THE SOFTWARE IS PROVIDED TO LICENSEE AT NO CHARGE ON AN AS-IS BASIS, FOR EVALUATION PURPOSES ONLY, AND NSB DISCLAIMS ALL WARRANTIES, WHETHER EXPRESS OR IMPLIED, INCLUDING, WITHOUT LIMITATION, ALL WARRANTIES OF TITLE, MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND NON-INFRINGEMENT.</li>
<li><strong>Limited Liability:</strong> IN NO EVENT SHALL NSB BE LIABLE FOR ANY DAMAGES WHATSOEVER (INCLUDING, WITHOUT LIMITATION, DIRECT, INDIRECT, CONSEQUENTIAL, SPECIAL, EXEMPLARY, PUNITIVE OR INCIDENTAL DAMAGES, OR DAMAGES FOR LOSS OF BUSINESS PROFITS, BUSINESS INTERRUPTION, LOSS OF BUSINESS INFORMATION, LOSS OF GOODWILL OR OTHER PECUNIARY LOSS) ARISING OUT OF THE USE OF OR INABILITY TO USE THE SOFTWARE, EVEN IF NSB HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES. SOME STATES MAY NOT ALLOW THE LIMITATION OR EXCLUSION OF LIABILITY FOR INCIDENTAL OR CONSEQUENTIAL DAMAGES, SO THE ABOVE LIMITATION OR EXCLUSION MAY NOT APPLY TO LICENSEE.</li>
<li><strong>Government Use:</strong> If Licensee is part of an agency, department, or other entity of the United States Government (&quot;<b>Government</b>&quot;), the use, duplication, reproduction, release, modification, disclosure or transfer of the Software is restricted in accordance with the Federal Acquisition Regulations as applied to civilian agencies and the Defense Federal Acquisition Regulation Supplement as applied to military agencies. The Software is a &quot;commercial item&quot;, &quot;commercial computer software&quot; and &quot;commercial computer software documentation&quot;. In accordance with such provisions, any use of the Software by the Government shall be governed solely by the terms of this Agreement.</li>
<li><strong>Governing Law/Jurisdiction:</strong> This Agreement is made in, and the validity, interpretation and enforcement of this Agreement shall be governed by and construed in accordance with the laws of, the State of New York, NY, excluding its conflicts of laws principles and excluding the United Nations Convention on Contracts for the International Sale of Goods.  All disputes arising out of or relating to this Agreement shall be submitted to the exclusive jurisdiction of the courts of competent jurisdiction located in the state of New York, NY, and each party irrevocably consents to such personal jurisdiction and waives all objections to such venue.</li>
<li><strong>Miscellaneous:</strong> This Agreement represents the complete agreement concerning the Software between NSB and Licensee and supersedes all prior agreements and representations between NSB and Licensee.  If any provision of this Agreement is held to be unenforceable for any reason, such provision shall be reformed only to the extent necessary to make it enforceable. Any waiver of any provision of this Agreement will be effective only if in writing and signed by NSB. The terms of this Agreement are personal to Licensee and may not be assigned or transferred for any reason whatsoever without the consent of NSB and any action or conduct in violation of the foregoing shall be void and without effect. NSB expressly reserves the right to assign this Agreement and to delegate any of its obligations hereunder.</li>
</ol>
<p>YOU EXPRESSLY ACKNOWLEDGE THAT YOU HAVE READ THESE TERMS AND UNDERSTAND THE RIGHTS, OBLIGATIONS, TERMS AND CONDITIONS SET FORTH HEREIN. BY CLICKING ON THE AGREE BUTTON AND/OR CONTINUING TO INSTALL OR USE THE SOFTWARE, YOU EXPRESSLY CONSENT TO BE BOUND BY THESE TERMS.</p>

</body>
</html>